lib/constants.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
For another approach, you could do:

```js
const firstBoardSpace = 0,
    lastBoardSpace = 23,
    playerOneStartSpace = firstBoardSpace - 1,
    playerOneEndSpace = lastBoardSpace + 1;

module.exports = {
    NUMBER_OF_GAME_PIECES: 15,
    NUMBER_OF_BOARD_SPACES: lastBoardSpace + 1,
    PLAYER_ONE_START_SPACE: playerOneStartSpace,
    PLAYER_ONE_END_SPACE: playerOneEndSpace,
    PLAYER_TWO_START_SPACE: playerOneEndSpace,
    PLAYER_TWO_END_SPACE: playerOneStartSpace,
    FIRST_BOARD_SPACE: firstBoardSpace,
    LAST_BOARD_SPACE: lastBoardSpace
};
```

The benefit of this is that there's less duplication. It's also obvious to a reader that it's intentional that player one's start space is player two's end space and vice versa. If those two values just happen to be the same number, it could be a coincidence.